packages/interaction-kit/src/helpers/arrays.ts

```diff
@@ -0,0 +1,14 @@
+type PrimitiveArray = Array<string | number | boolean | null | undefined>;
```

Comment:
```suggestion
type PrimitiveArray = Array<unknown>;
```

Reply:
I want to explicitly disallow objects since this won't accurately compare them